src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.html

```diff
@@ -40,136 +60,148 @@ <h4>{{ 'arTeMiSApp.tutorExerciseDashboard.gradingInstructions' | translate }}</h
+                                {{ calculateStatus(submission) }}
+                            </td>
+                            <td>
+                                <span *ngIf="submission.result !== null && submission.result.score !== null">{{ submission.result.score }} %</span>
+                            </td>
+                            <td>
+                                <button
+                                    *ngIf="submission !== null && calculateStatus(submission) === 'DRAFT'; else continueButton"
+                                    (click)="openAssessmentEditor(submission.id)"
+                                    class="btn btn-warning btn-sm"
+                                >
+                                    <fa-icon icon="folder-open" [fixedWidth]="true"></fa-icon>&nbsp;{{ 'arTeMiSApp.tutorExerciseDashboard.continueAssessment' | translate }}
+                                </button>
+                                <ng-template #continueButton>
+                                    <button
+                                        *ngIf="submission !== null && submission.result !== null"
+                                        (click)="openAssessmentEditor(submission.id)"
+                                        class="btn btn-primary btn-sm"
+                                    >
+                                        <fa-icon icon="folder-open" [fixedWidth]="true"></fa-icon>&nbsp;{{ 'arTeMiSApp.tutorExerciseDashboard.openAssessment' | translate }}
+                                    </button>
+                                </ng-template>
+                            </td>
+                        </tr>
+                        <tr *ngIf="unassessedSubmission && unassessedSubmission?.id > 0">
+                            <td></td>
+                            <td>{{ unassessedSubmission?.submissionDate | date: 'MMM d, y HH:mm:ss' }}</td>
+                            <td></td>
+                            <td>{{ 'arTeMiSApp.tutorExerciseDashboard.new' | translate }}</td>
+                            <td></td>
+                            <td>
+                                <button class="btn btn-success btn-sm" (click)="openAssessmentEditor(unassessedSubmission.id, true)">
+                                    <fa-icon icon="folder-open" [fixedWidth]="true"></fa-icon>{{ 'arTeMiSApp.tutorExerciseDashboard.startAssessment' | translate }}
+                                </button>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+
+                <ng-template #noSubmissions>
+                    <div class="alert alert-info">
+                        {{ 'arTeMiSApp.tutorExerciseDashboard.noSubmissions' | translate }}
+                    </div>
+                </ng-template>
             </div>
-        </ng-template>
+        </div>
+
+        <jhi-assessment-instructions
+            class="max-vh-100" style="max-height: 100vh"
```

Comment:
is the inline style necessary or does it only repeats the class?
nevertheless we should avoid inline styles as much as possible